interface/modules/custom_modules/oe-module-dorn/public/orders.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once "../../../../globals.php";
+
+ use OpenEMR\Common\Acl\AclMain;
+ use OpenEMR\Common\Csrf\CsrfUtils;
+ use OpenEMR\Common\Twig\TwigContainer;
+ use OpenEMR\Modules\Dorn\ConnectorApi;
+ use OpenEMR\Core\Header; //this is needed along with setupHeader() to get the pop up to appear
+
+$tab = "orders";
+$pageTitle = "DORN Orders";
```

Comment:
Doing above shortcut will bypass the translation pipeline (we run a script every once in while to collect new translations) which needs them to be in a translation function. A quick work around is to just make a comment here with:
```
// for translation collection pipeline
// xl('DORN Orders')
```

Reply:
done